src/App.js

```diff
@@ -1,25 +1,39 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import Header from "./components/Header";
+import Products from "./components/Products";
+import Cart from "./components/Cart";
+
+const App = () => {
+  const [page, setPage] = useState("products");
+  const [selectedcat, setselectedcat] = useState("");
+  const [cart, setCart] = useState([]);
+  const [category, setcategory] = useState([]);
```

Comment:
improve naming -> better would be 
```suggestion
  const [page, setPage] = useState("products");
  const [selectedCategory, setSelectedCategory] = useState("");
  const [cart, setCart] = useState([]);
  const [categories, setCategories] = useState([]);
```

Reply:
always follow camel case while naming functions and variabled in react